### PR TITLE
[Check out Skin Changing First Please] Title Card with Tagline and GitHub Link

### LINKED ISSUE #58: "Informative welcome div about the site and project"
"Should have some info on what site you're on - you know, information and the likes."

Quotes taken directly from @leafoflegend.

### LINKED ISSUE #68: Need some sort of tutorial or guidance for how to interact in world
Perhaps this tutorial can appear during the initial load and the spinner turns into an "I'm Ready!" button or some such thing.

### PR DESCRIPTION
**Pull request by bushidocodes**
Created a title card with links to GitHub and ported it back to the Physical DOM with a spinner for progressive enhancement while the initial ~4mb bundle.js file downloads.

This is built on my skin changing PR, so I'll have to merge any fixes there into here before merging.

Resolves #58 
Resolves #68

### CODE REVIEW

**Review by parky22 — APPROVED** [review]

File: browser/react/index.js [hunk 1]
```diff
@@ -27,9 +27,9 @@ const onHomeEnter = () => {
     store.dispatch(whoami())
       .then(() => {
         const user = store.getState().auth;
-        console.log(`onHomeEnter is comparing on ${user}`);
+        // console.log(`onHomeEnter is comparing on ${user}`);
```
Comment (bethqiang) [inline browser/react/index.js]: Do we need most of these console.logs anymore? If not, can we delete them?

File: browser/redux/reducers/auth.js [hunk 2]
```diff
@@ -18,23 +18,23 @@ export const authenticated = user => ({
 });
 
 export const login = (username, password) => {
-  console.log(`Logging in with ${username}, ${password}`);
+  // console.log(`Logging in with ${username}, ${password}`);
```
Comment (bethqiang) [inline browser/redux/reducers/auth.js]: Same with these. 😄

File: browser/react/components/Login/Title.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <h1 style={props.styles.appTitle}>Transcend</h1>
+    <div style={props.styles.subtitleCenter}> Virtual Teams </div>
+    <div style={props.styles.subtitleCenter}> Shared VR Experiences </div>
+    <div style={props.styles.subtitleCenter}> Real Friends </div>
+    <a target="_blank" href="https://github.com/TranscendVR/transcend" style={props.styles.viewOnGitHub}>
+      <span className="fa fa-github" style={props.styles.viewOnGitHubIcon}></span>
+      View on GitHub
+    </a>
+  </div>
+);
```
Comment (bethqiang) [inline browser/react/components/Login/Title.js:16]: I'm probably missing something relatively obvious here/maybe wasn't paying attention to Slack convos, but what's the difference between this and the stuff in the actual HTML? (And likewise, the difference between the spinny thingy below and our React/Material/no-offense-slightly-prettier spinny thingy?)

**Review by bethqiang — COMMENTED** [review]
TL;DR: console.logs & I'm confused.

Also, I noticed in the screenshot you posted that the transition/fade-to-transparent-with-white-border-and-text on the text fields went away... 😢  (This might have gone missing earlier, but I just noticed...)

I do like the div on top though! It looks good! 😄
